base/settings/tests/testthat/test.addSecrets.R

```diff
@@ -35,32 +35,3 @@ test_that("`addSecrets` adds secret settings when force is TRUE and secrets have
   expect_equal(updated_settings$database$section$name, "pecan")
   expect_equal(updated_settings$database$section$password, "pecan")  
 })
-
-test_that("`addSecrets` adds secret settings when force is FALSE and secrets have not been added", {
```

Comment:
Rather than removing the test it should be updated according to the latest changes in the `addSecrets` function.

Reply:
If the scenario this test checks is completely purged and will not occur, only then it seems fair to delete a test.

Reply:
Fixed